helm/alfresco-content-services/README.md

```diff
@@ -60,6 +60,7 @@ Hence, setting up explicit Container memory and then assigning a percentage of i
 | alfresco-digital-workspace.enabled | bool | `true` |  |
 | alfresco-digital-workspace.env.API_URL | string | `"{protocol}//{hostname}{:port}"` |  |
 | alfresco-digital-workspace.env.APP_CONFIG_AUTH_TYPE | string | `"BASIC"` |  |
+| alfresco-digital-workspace.extraEnv | string | `"{{- if .Values.ai.enabled }}\n- name: APP_CONFIG_PLUGIN_AI_SERVICE\n  value: 'ai.enabled'\n{{- end }}"` |  |
```

Comment:
Are you happy with this appear as:
`{{- if .Values.ai.enabled }}\n- name: APP_CONFIG_PLUGIN_AI_SERVICE\n value: 'ai.enabled'\n{{- end }}`
in the readme? With the `\n`?

Reply:
@montgolfiere @PunchedChimera FYI, we have just introduced helm-docs pre-commit hook on master to generate the configuration documentation from the values file.

I would recommend pulling master into your branch, adding the comment to the values.yaml file (you'll see examples in there) and running the helm-docs pre-commit. Sergiu should be able to help you with this if you run into problems.

Reply:
It was generated that way by the pre-commit hooks. But something is not quite right.

We'd need to compare to similar examples here:

https://github.com/Alfresco/alfresco-process-infrastructure-deployment/blob/develop/helm/alfresco-process-infrastructure/values.yaml#L695-L698

Reply:
He is still just trying to get it to work at this point and will probably change it after

Reply:
Ah OK, ignore me then in that case :-)

Reply:
Gavin may have even been replying to me who also added a comment at the same time but then deleted it, just to confuse things 😆

Reply:
I think I was replying to @montgolfiere anyway, seems to be sorted now :-)